qualtran/bloqs/state_preparation/state_preparation_via_rotation_test.py

```diff
@@ -64,8 +64,9 @@ def test_state_prep_via_rotation(bloq_autotester):
     ],
 )
 def test_exact_state_prep_via_rotation_(phase_bitsize: int, state_coefs: Tuple[complex, ...]):
+    # https://github.com/python/mypy/issues/5313
```

Comment:
we make heavy use of attrs converters, so this is one to look out for. A lot of places, our classes need a sequence of something and need to be hashable, so we convert to `Tuple[T, ...]` and I've noticed its very hard to get the Pycharm type checker and pyright (which are the two I've played around with before) to accept this.

Reply:
Yeah, this is annoying that mypy craps out with this.  It looks like it doesn't understand the auto-converter.  It looked fine to me, so I have just been type: ignoring these.